rotkehlchen/chain/ethereum/modules/balancer/balancer.py

```diff
@@ -105,7 +105,21 @@ def __init__(
         self.msg_aggregator = msg_aggregator
         self.trades_lock = Semaphore()
         try:
+            # TODO: add more flexibility on the module/endpoints availability
```

Comment:
Before using 2 subgraphs it was trivial when map this module to "None" (in manager.py after catching `ModuleInitializationFailure`). Now I'm not that sure how to deal with it.

Reply:
we discussed in discord that this is probaly fine for now